### DISCUSSION TITLE
Multiple columns sorting allowed

### DISCUSSION THREAD

**Discussion by HGZdev** [community/community#10626] [Projects and Issues] [answered]
It would be extremely useful to sort issues by a few columns at the same time.

**Comment by chrisglein** [comment]
Yes please!

Scenario: I want a table view where I can sort by assignee first and by a custom iteration field second. That allows me to quickly view the sequence of everyone's flow of work but in a table view.
Scenario: Sort by priority first and by status second, helping me catch high priority issues that aren't getting traction.

Frankly, there are a ton of scenarios for this. As is it's becoming essential to export (#6584) and then do all this in Excel. If you're doing significant munging of the data, sure. But that's overkill for a 2 tiered sort. 

FYI, something that's sometimes an option is using Group By on one field and then Sort By on a secondary field.

**Reply by cschreib** [reply]
> FYI, something that's sometimes an option is using Group By on one field and then Sort By on a secondary field.

Except, to my knowledge, you cannot choose the ascending/descending sorting order for "Group By" (https://github.com/github/feedback/discussions/13842). So right now, I've got "Group By" priority, and the low priority items are on top :|

**Comment by ccolella-mdc** [comment]
Sorting by multiple columns would be very useful, but just being able to choose the order of the Group By would solve a lot of the use cases of multiple sorting, I think.

**Comment by galah92** [comment]
It would be very useful for me too.

**Comment by rileybroughten** [comment, marked as answer]
Closing this duplicate in favor of the top-voted discussion post https://github.com/orgs/community/discussions/8895 for this request.

Keep an eye on that post for updates! ✨
